src/utils/connectToRedis.ts

```diff
@@ -1,11 +1,27 @@
+import fs from 'fs';
 import Ioredis from 'ioredis';
-import { defaultTo, once } from 'lodash';
+import { once } from 'lodash';
 import config from '../config';
 import logger from '../logger';
 
 export default once((): (() => Promise<Ioredis.Redis>) => {
   return once(async () => {
     logger.info('Creating redis connection');
-    return new Ioredis(defaultTo(config.redis.url, 'redis://127.0.0.1:6379/0'));
+
+    if (config.redis.tlsIsEnabled) {
+      return new Ioredis({
+        db: config.redis.db,
+        password: config.redis.password,
+        host: config.redis.host,
+        port: config.redis.port,
+        tls: {
+          cert: fs.readFileSync('./certificates/redis/redis.crt'),
+          key: fs.readFileSync('./certificates/redis/redis.key'),
+          ca: fs.readFileSync('./certificates/redis/ca.crt'),
+        },
+      });
+    }
```

Comment:
If TLS is not enabled, what's the difference between returning
new Ioredis({
        db: config.redis.db,
        password: config.redis.password,
        host: config.redis.host,
        port: config.redis.port,
      });
vs
new Ioredis(config.redis.url);

Reply:
Hey @lucuilearningpool 👋  No difference, this is the same